### PR TITLE
[CMR-81]Feature/filter years land use

### PR DESCRIPTION
**Pull request by GabrielMoreschi**
[CMR-81] - Land-use filter:

Foi adicionado o filtro por ano de acordo com os filtros de CR e TI.

### CODE REVIEW

File: store/land-use.js [hunk 1]
```diff
@@ -152,35 +158,49 @@ export const actions = {
...
+        }
+
+        const years = await this.$api.$get('land-use/years/', { params })
+
+        if (years) {
+            commit('setFilterOptions', {
+                ...state.filterOptions,
+                year: years
+        })
+    }
```
Comment (dagnaldo) [inline store/land-use.js:201]: What is this identation?

File: store/land-use.js [hunk 2]
```diff
@@ -152,35 +158,49 @@ export const actions = {
...
     },
+
     async getTiOptions({ commit, state }, cr) {
         const params = {
             co_cr: cr.toString(),
         }
 
         const tis = await this.$api.$get('funai/ti/', { params })
-
+
```
Comment (dagnaldo) [inline store/land-use.js:181]: identation?

File: components/land-use/LandUseFilter.vue [hunk 3]
```diff
@@ -270,6 +260,11 @@ export default {
             else this.filters.ti = null
         },
 
+        populateYearsOptions(ti) {
+            if (ti) this.$store.dispatch('land-use/getYearsOptions', ti)
```
Comment (dagnaldo) [inline components/land-use/LandUseFilter.vue:264]: Where is the loader?

Comment (GabrielMoreschi) [reply components/land-use/LandUseFilter.vue:264]: Resolved

File: store/land-use.js (hunk 2 above)
Comment (GabrielMoreschi) [reply store/land-use.js:181]: Resolved

File: components/land-use/LandUseFilter.vue [hunk 4]
```diff
@@ -221,16 +215,28 @@ export default {
                 ti: null,
             },
             isLoadingTotal: false,
+            loading: false,
```
Comment (dagnaldo) [inline components/land-use/LandUseFilter.vue:218]: Loading what?

**Review by dagnaldo — CHANGES_REQUESTED** [review]
More descriptive variables.

File: components/land-use/LandUseFilter.vue (hunk 4 above)
Comment (GabrielMoreschi) [reply components/land-use/LandUseFilter.vue]: Resolved

File: components/land-use/LandUseFilter.vue [hunk 5]
```diff
@@ -221,16 +218,28 @@ export default {
                 ti: null,
             },
             isLoadingTotal: false,
+            loadingYears: false,
```
Comment (dagnaldo) [inline components/land-use/LandUseFilter.vue:221]: isLoadingYears... 
Please follow the pattern.

**Review by dagnaldo — CHANGES_REQUESTED** [review]

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**